Detectors/GlobalTracking/src/MatchTPCITS.cxx

```diff
@@ -1336,7 +1336,7 @@ bool MatchTPCITS::refitTrackTPCITS(int iTPC, int& iITS)
     auto posStart = tracOut.getXYZGlo();
     auto tImposed = timeC * mTPCTBinMUSInv;
     if (std::abs(tImposed - mTPCTracksArray[tTPC.sourceID].getTime0()) > 550) {
-      LOGP(error, "Impossible imposed timebin {} for TPC track time0:{}, dBwd:{} dFwd:{} TB | ZShift:{}, TShift:{} | Trc: {}", tImposed, mTPCTracksArray[tTPC.sourceID].getTime0(),
+      LOGP(alarm, "Impossible imposed timebin {} for TPC track time0:{}, dBwd:{} dFwd:{} TB | ZShift:{}, TShift:{} | Trc: {}", tImposed, mTPCTracksArray[tTPC.sourceID].getTime0(),
```

Comment:
we could leave only the first line alarm, and make the rest info?

Reply:
True, will change that